.github/ISSUE_TEMPLATE/general-issue-form.yaml

```diff
@@ -51,10 +51,14 @@ body:
     id: logs
     attributes:
       label: Relevant log output
-      placeholder: |
+      description: |
         :checkhealth
         :messages
         :e $LUNARVIM_CACHE/DIR/lvim.log
+      placeholder: |
+      ```
+      paste here
+      ```
```

Comment:
doesn't this need to be indented?

> There is a problem with this template

>
> YAML syntax error: (): found character that cannot start any token while scanning for the next token at line 59 column 7. [Learn more](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms).

Reply:
true, I'll try to fix it.
What do you think about splitting this into several inputs? one for each command, so people don't need to provide their own backtics?

Reply:
good idea!